bedrock/firefox/templates/firefox/new.html

```diff
@@ -130,8 +135,13 @@ <h1 class="large">
               <li>
                 <a href="{{ url('firefox.os.index') }}">{{_('Learn about Firefox OS')}}</a>
               </li>
+                {% if not l10n_has_tag('ios_announce') %}
```

Comment:
Above you simply removed the message about Firefox being unsupported. Should you do the same here instead of checking for the lack of tag?